package.json

```diff
@@ -16,12 +16,12 @@
     "pouchdb"
   ],
   "dependencies": {
-    "pouchdb-upsert": "^1.0.2",
-    "pouchdb-collections": "^1.0.0",
     "argsarray": "0.0.1",
     "bluebird": "^1.2.4",
+    "debug": "git://github.com/visionmedia/debug.git",
```

Comment:
having to do this is annoying, but I'd prefer not to be held up by another project